### ISSUE TITLE
There may be something wrong with the `mmap` implementation

### ISSUE THREAD

**Issue by bigboss2063** [nutsdb/nutsdb#551]
When using mmap's read and write mode, read performance will be very poor, about ten times slower than file IO.

I think there may be a problem with the mmap library.

**Comment by dxyinme** [comment]
I think github.com/xujiajun/mmap-go v1.0.1 is too old. maybe we need let xujiajun/mmap-go sync with github.com/edsrzf/mmap-go latest.

**Comment by bigboss2063** [comment]
You're absolutely right; it can even be directly replaced with github.com/edsrzf/mmap-go. Do you want to do this? 😊

**Comment by dxyinme** [comment]
let me have a try.

**Comment by dxyinme** [comment]
> Do you wish to continue maintaining NutsDB in the future? For example, fixing some bugs in the Bucket Manager, improving transactions, optimizing memory usage, and so on.
> […](#)

of cause I will, that's my pleasure for me. 😸

**Comment by dxyinme** [comment]
我想这个issue可以close，然后kvstore-bench那边的PR也帮忙看一眼 @bigboss2063

**Comment by bigboss2063** [comment]
okok @dxyinme
